packages/framer-motion/src/animation/GroupPlaybackControls.ts

```diff
@@ -0,0 +1,29 @@
+import { AnimationPlaybackControls } from "./types"
+
+export class GroupPlaybackControls implements AnimationPlaybackControls {
+    animations: AnimationPlaybackControls[]
+
+    constructor(animations: Array<AnimationPlaybackControls | undefined>) {
+        this.animations = animations.filter(
+            Boolean
+        ) as AnimationPlaybackControls[]
```

Comment:
I'm still so disappointed that typescript can't figure this out 😞 
Is there no way to go without the cast?

Reply:
It actually didn't even work with a more explicit function either (that checked specficially for undefined)